frontend/pages/index.tsx

```diff
@@ -2,55 +2,23 @@ import type { NextPage } from 'next';
 import Head from 'next/head';
```

Comment:
What the difference between `index.tsx` and `home.tsx`? 
If they are the same, you can only push one.

Reply:
OK